lighthouse-core/test/audits/dobetterweb/no-datenow-test.js

```diff
@@ -66,4 +66,21 @@ describe('Page does not use Date.now()', () => {
     assert.equal(auditResult.rawValue, false);
     assert.equal(auditResult.extendedInfo.value.length, 2);
   });
+
+  it('fails when usage hasn`t url', () => {
+    const auditResult = DateNowUseAudit.audit({
+      DateNowUse: {
+        usage: [
+          {line: 1, col: 1},
+          {url: 'http://example.com/two', line: 10, col: 1},
+          {url: 'http://example2.com/two', line: 2, col: 22}
+        ]
+      },
+      URL: {finalUrl: URL},
+    });
+
+    assert.equal(auditResult.rawValue, false);
+    assert.ok(auditResult.extendedInfo.value[0].url === '');
+    assert.equal(auditResult.extendedInfo.value.length, 2);
```

Comment:
Can you check that `extendedInfo.value` has a url and is what you expect? From the first assert, it looks like `extendedInfo.value` will be the first to values in `DateNowUse.usage`?